scripts/verify_env.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+"""
+Used to check missing deployed lambda env variables with local environment variables.
+Does not check the values of the variables, limited to a check if the key is present.
+
+"""
+import os
+import boto3
+import json
+
+
+lambda_client = boto3.client("lambda")
+
+
+def get_local_lambda_environment_keys():
+    env = dict()
+    deployment_env = ["ADMIN_USER_EMAILS", "DSS_ES_ENDPOINT",
+                      "GOOGLE_APPLICATION_CREDENTIALS", "GOOGLE_APPLICATION_SECRETS"]
+    deployment_env += os.environ['EXPORT_ENV_VARS_TO_LAMBDA'].split()
+    for name in deployment_env:
+        try:
+            env[name] = os.environ[name]
+        except KeyError:
+            print(f"Warning: {name} not defined")
+    return env.keys()
+
+
+def get_lambda_environment(stage):
+    parms = lambda_client.get_function_configuration(
+        FunctionName=f"dss-{stage}"
```

Comment:
This only checks `f"dss-{stage}"`, should it check the other lambdas as well?

Reply:
removed lambda checks, went to SSM instead.